src/Microsoft.Health.Dicom.Functions/DurableFunctionsCleanup.cs

```diff
@@ -0,0 +1,73 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using EnsureThat;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using System.Threading;
+using System;
+using Microsoft.Health.Dicom.Functions.Indexing.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Health.Dicom.Functions
+{
+    /// <summary>
+    /// Represents the Azure Functions that perform the cleanup of completed Durable Functions used for re-indexing.
+    /// </summary>
+    public class DurableFunctionsCleanup
+    {
+
+        private readonly ReindexConfiguration _clientConfig;
+        // TODO: Leverages services for fulfilling the various activies
+        public DurableFunctionsCleanup(IOptions<IndexingConfiguration> configOptions)
+        {
+            EnsureArg.IsNotNull(configOptions, nameof(configOptions));
+            _clientConfig = configOptions.Value.Add;
+        }
+
+        // Currently an HTTP trigger, TODO: change to the appropriate trigger.
+        [FunctionName("DurableFunctionsCleanup")]
+        public async Task<String> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "get", "post", Route = null)] HttpRequest req,
+            [DurableClient] IDurableOrchestrationClient client,
+            ILogger log,
+            CancellationToken hostCancellationToken)
+        {
+            EnsureArg.IsNotNull(req, nameof(req));
+            using var cancellationTokenSource = CancellationTokenSource.CreateLinkedTokenSource(hostCancellationToken, req.HttpContext.RequestAborted);
+
+            log.LogInformation("Durable Functions cleanup started.");
+
+            EnsureArg.IsNotNull(client, nameof(client));
+
+            Console.WriteLine(_clientConfig.IsPurgeErrorEnabled);
```

Comment:
I know this is for testing, but the logger should help here too by printing to console